src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Fluent/Resources/Theme/Dark.xaml

```diff
@@ -440,6 +440,51 @@
     <SolidColorBrush x:Key="ContextMenuForeground" Color="{StaticResource TextFillColorPrimary}" />
 
     <!--  DataGrid  -->
+    <SolidColorBrush x:Key="DataGridColumnFloatingHeaderBorderBrush" Color="{StaticResource ControlStrongStrokeColorDefault}" />
+    <SolidColorBrush x:Key="DataGridHeaderDropSeparatorBackground" Color="{StaticResource ControlStrongStrokeColorDefault}" />
+    <SolidColorBrush x:Key="DatatGridSelectAllButtonForeground" Color="{StaticResource TextFillColorPrimary}" />
+    <SolidColorBrush x:Key="DataGridCellFocusVisualBorderBrush" Color="{StaticResource TextFillColorSecondary}" />
+    <SolidColorBrush x:Key="DataGridPillFillBrush" Color="{StaticResource SystemAccentColorDark1}" />
```

Comment:
![image](https://github.com/user-attachments/assets/c093e029-a605-446b-a656-4ba27fef3dbd)

DataGrid vs. WPF Gallery.

Not sure if this is the right accent color? This might not meet a11y requirements in dark mode,